common/util_test.go

```diff
@@ -213,6 +213,14 @@ func TestCreateHistoryStartWorkflowRequest_ExpirationTimeWithCron(t *testing.T)
 // with & without cron, delayStart and jitterStart.
 // - Also see tests in cron_test.go for more exhaustive testing.
 func TestFirstDecisionTaskBackoffDuringStartWorkflow(t *testing.T) {
+	firstRunTimestampPast, _ := time.Parse(time.RFC3339, "2017-12-17T08:00:00+00:00")
+	firstRunTimestampFuture, _ := time.Parse(time.RFC3339, "2018-12-17T08:00:02+00:00")
+	firstRunAtTimestampMap := map[string]int64{
```

Comment:
Instead of defining this new map and doing a combination of this and the test cases we can just update the `tests` to enumerate all cases with and without firstRunat. This will also simplify the validation logic I think

Reply:
Thanks, I have made the responding changes.